### PR TITLE
test file added for Sharding Insert Values Token

### LINKED ISSUE #16100: Add unit test for ShardingInsertValuesToken#toString
Hi community,

This issue is for [OpenForce](https://github.com/apache/shardingsphere/discussions/12683#discussioncomment-2139441)

### PR DESCRIPTION
**Pull request by shreya024**
Fixes #16100 .

### CODE REVIEW

**Comment by shreya024** [comment]
CI is failing because of error in shardingsphere-pipeline-test

File: shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/ShardingInsertValuesTokenTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+public final class ShardingInsertValuesTokenTest {
+
+    private ShardingInsertValuesToken shardingInsertValuesToken;
+
+    private RouteUnit routeUnit;
+
+    @Before
+    public void setup() {
+        shardingInsertValuesToken = new ShardingInsertValuesToken(0, 2);
+        final RouteMapper routeMapper = new RouteMapper("logic_ds", "actual_ds");
```
Comment (strongduanmu) [inline shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/ShardingInsertValuesTokenTest.java:47]: @shreya024 Please remove final for params in method scope.

File: shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/ShardingInsertValuesTokenTest.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        final RouteMapper routeMapper1 = new RouteMapper("tbl", "tbl_0");
+        final RouteMapper routeMapper2 = new RouteMapper("tbl", "tbl_1");
+        routeUnit = new RouteUnit(routeMapper, Arrays.asList(routeMapper1, routeMapper2));
+        ExpressionSegment expressionSegment1 = new LiteralExpressionSegment(0, 0, "shardingsphere");
+        ExpressionSegment expressionSegment2 = new LiteralExpressionSegment(0, 0, "test");
+        List<ExpressionSegment> expressionSegment = new ArrayList<>(2);
```
Comment (strongduanmu) [inline shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/test/java/org/apache/shardingsphere/sharding/rewrite/token/pojo/ShardingInsertValuesTokenTest.java:53]: `new LinkedList<>();` may be better.

**Review by strongduanmu — CHANGES_REQUESTED** [review]

**Review by strongduanmu — APPROVED** [review]
@shreya024 Thank you.

**Pull request merged**